### PR TITLE
Added option to change the text styling

### PR DESCRIPTION
**Pull request by sromku**

### CODE REVIEW

**Comment by sromku** [comment]
@dvirsky documentation was added

File: README.md [hunk 1]
```diff
@@ -57,3 +57,49 @@ Then you can either embed an `ExplainView` right in any activity, or launch an a
 
 In Android Studio - just clone this project, import it into Android Studio, and add it as a dependency to your app.
 
+
+## Styling
+
+It is possible tp customize and apply your own style theme to the texts.
```
Comment (arikfr) [inline README.md]: typo: "tp" -> "to".

Comment (sromku) [inline README.md]: @arikfr right. fixed

**Comment by sromku** [comment]
@arikfr fixed typo

**Comment by dvirsky** [comment]
@sromku I'm merging this but in your free time, please add some unit tests for the styling? thanks.

**Pull request merged**